include/swift/AST/DiagnosticsSema.def

```diff
@@ -98,6 +98,9 @@ ERROR(could_not_use_type_member,none,
 ERROR(could_not_use_type_member_on_instance,none,
       "static member %1 cannot be used on instance of type %0",
       (Type, DeclName))
+ERROR(could_not_use_type_member_on_instance_without_name,none,
+      "static element %0 cannot be referenced as an instance member",
```

Comment:
Nit: `static member` is right here.

Reply:
While we’re at it, this doesn’t seem much clearer than what was there before

Reply:
Just following @jrose-apple's:

> The starter bug form of this would be to only include the name if the base expression was a simple DeclRefExpr, or perhaps also a MemberRefExpr

I'm open to suggestions

Reply:
IMO, it's clearer because it directly contrasts static with instance members without any extraneous information, but agree that `static member` is the correct term.

Reply:
Sorry, my "only" was ambiguous! I meant "this should be checked in all circumstances, but don't worry about printing the name of the member if it's not easy to do so". I like Huon's suggested wording in the bug report.